UnityClient/Assets/Scripts/CameraMouseControl.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraMouseControl : MonoBehaviour
+{
+
+    private Vector2 mouseDirection;
```

Comment:
Please prefix private fields with an underscore for uniformity. Example: `_mouseDirection`.